### ISSUE TITLE
Online test

### ISSUE THREAD

**Issue by remicollet** [nette/utils#112]
Followin #91 

We still encounter failure without internet access...

```
-- FAILED: Nette\Utils\FileSystem basic usage. | tests/Utils/FileSystem.phpt
   Exited with error code 255 (expected 0)
   Nette\IOException: Unable to copy file 'http://example.com' to '/dev/shm/BUILD/utils-f6586f827292bd35c8593df943437f2247ba5337/tests/tmp/0.95770311730773/3/x/y/file'.
   
   in Nette/Utils/FileSystem.php(61) 
   in tests/Utils/FileSystem.phpt(44) Nette\Utils\FileSystem::copy()
   in utils-f6586f827292bd35c8593df943437f2247ba5337/tests/bootstrap.php(39) {closure}()
   in tests/Utils/FileSystem.phpt(71) test()

```

**Comment by remicollet** [comment]
Same patch still apply and fix the issue.

**Comment by dg** [comment]
Should be closed by #93
